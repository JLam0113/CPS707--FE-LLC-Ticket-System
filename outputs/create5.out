Welcome, please enter a command.
Please enter your username
Privileged Login successful, now logged in as admin, please enter a command
Create account selected, please enter the username:
Please enter the account type:
Please enter the initial credit amount:
Transaction unsuccessful (credit exceeded $999,999 or was less than 0), please enter a command.
Session ended
