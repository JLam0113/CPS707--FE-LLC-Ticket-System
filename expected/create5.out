Welcome, please enter a command.
Please enter your username
Privileged Login successful, now logged in as admin, please enter a command
Create account selected, please enter the username:
Please enter the account type:
Invalid credit (exceeds 999,999). Please enter a command.
Session ended
